WardReports/Generate_weekly_ward_report_scripts.py

```diff
@@ -0,0 +1,67 @@
+import pandas as pd
+import numpy as np
+
+import datetime as dt
+import math
+
+import civis
+import os
+
+from helpers import *
+
+client = civis.APIClient()
+
+def main():
+    #actual ward table import to be used once testing is complete
+    '''
+    ward_email_data = civis.io.read_civis(database='City of Chicago',
+                                          table = 'scratch.ward_office_info',
+                                          use_pandas = True)
+                                          '''
```

Comment:
@srao-civis  I created a table called cic.ward_office_info that has the actual email addresses we should use. lets test the code to make sure it works with that table, while commenting the line at the bottom that actually sends the emails